lib/Client/NoOpSpan.php

```diff
@@ -6,6 +6,8 @@
 
 class NoOpSpan implements \LightStepBase\Span {
     public function guid() { return ""; }
+    public function traceGUID() { return ""; }
```

Comment:
needs to match parent span in order to be used interchangeably with the ClientSpan